pkg/controller/controller.go

```diff
@@ -102,8 +105,28 @@ func NewController(clientset kubernetes.Interface, ssclientset ssclientset.Inter
 		},
 	})
 
+	infsecret := informers.NewSharedInformerFactory(clientset, 0).Core().V1().Secrets().Informer()
```

Comment:
Why are we creating this informer in a different way as the Sealed Secret informer?

Reply:
it is not the same informer. This is an informer generated directly by the client-go API. This is why I needed to include the library.